Wrapping/Python/SimpleITK/extra.py

```diff
@@ -298,4 +300,98 @@ def GetImageFromArray(arr, isVector=None):
     return img
 
 
-__all__ = ["Resample", "GetArrayViewFromImage", "GetArrayFromImage", "GetImageFromArray"]
+def ReadImage(fileName: Union[str, List[str]],
+              outputPixelType: "PixelIDValueEnum" = sitkUnknown,
+              imageIO: str = "") \
+        -> Image:
+    r"""ReadImage is a procedural interface to the ImageFileReader class which is convenient for most image reading
+     tasks.
+
+    This method can read a single image or a list of images into a volume.
+
+    Parameters
+    ----------
+    fileName
+     A single or a list of file names. the filename of an Image e.g. "cthead.mha"
+    outputPixelType
+     The pixel type of the returned Image. By default the value is sitkUnknown, which enable the output pixel type to
+     be same as the file. If the pixel type is specified then the itk::ConvertPixelBuffer will be used to convert the
+     pixels.
+    imageIO
```

Comment:
"By default the value is sitkUnknown, which enable the output pixel type to be same as the file." ->

"By default the value is sitkUnknown, which indicates that the output pixel type be the pixel type stored in the file."